src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -490,6 +490,86 @@
                 }
             ]
         },
+        {
+            "path": "/v1/debug/partitions/{namespace}/{topic}/{partition}/block_raft_op/{op}",
```

Comment:
I do not particularly like the name `block_raft_of`. This is still very specific operation that has nothing to do with the actual failure that may happen in production system

Reply:
`append_entries_error_injection` with values `on` and `off`?